schedule/ha/bv/migration/migration_verify_sles_ha.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: migration_verify_sles_ha
+description: >
+  Offline migration from installed SLES with HA extension
+
+  This test does an offline migration using the *DVD* media, with or without SCC registration.
+vars:
+  BOOT_HDD_IMAGE: "1"
+  DESKTOP: "textmode"
+  BOOTFROM: "c"
+  TIMEOUT_SCALE: "2"
+  QEMU_DISABLE_SNAPSHOTS: "1"
+  HA_CLUSTER: 1
```

Comment:
Maybe good to be coherent for each value, protected or not :) And if protected use `'1'`, as all current code in `schedule/` is like this (again more coherent IMO).